### PR TITLE
move euth ideaCollection to core

### PR DESCRIPTION
**Pull request by xi**
Rebased from #2492 after merge of #2489.

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/Workbench.ts [hunk 1]
```diff
@@ -192,6 +192,22 @@ export var registerRoutesFactory = (
...
+                return adhHttp.options(item.path).then((options : AdhHttp.IOptions) => {
+                    if (!options.POST) {
+                        throw 401;
+                    } else {
+                        return {
+                            proposalUrl: version.path
+                        };
+                    }
+                });
+            }])
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/Workbench.ts]: Not urgent, but: We could check `processOptions` and only register this view if needed.

**Comment by xi** [comment]
LGTM.

**Comment by nidico** [comment]
Is this already supposed to be complete and working? There's still a couple of references to `meinberlin` in the code.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/Workbench.ts (hunk 1 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/IdeaCollection/Workbench/Workbench.ts]: Nice idea. Maybe later.

**Pull request merged**